docs/api/wallet-rpc.yaml

```diff
@@ -1113,6 +1113,8 @@ components:
         destination:
           type: string
           example: bcrt1qu7k4dppungsqp95nwc7ansqs9m0z95h72j9mze
+        txfee:
+          type: integer
```

Comment:
could add an example here, but it's fine really. It's good though if the API docs emphasize the same (very non-obvious and pretty complicated) logic of how this number is interpreted, as is in the joinmarket.cfg comments.

Can I suggest adding something about it in the `description` field of `/wallet/{walletname}/taker/direct-send`? See e.g. `/maker/start` where we write some details similarly.

Reply:
Yes, could do it, was just lazy, wanted to make something working fast, as Jam would benefit from this.